### PR TITLE
feat/Upload and assign art to place

### PR DESCRIPTION
**Pull request by shea-hawkins**

### CODE REVIEW

**Comment by ccarey221** [comment]
🚢

**Comment by shea-hawkins** [comment]
The CircleCI tests are not yet setup, so they don't pass.

**Comment by ccarey221** [comment]
🚢

File: server/resources/place/placeModel.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = function Place(db, Sequelize) {
-  return db.define('Place', {
+  return PlaceModel =  db.define('Place', {
```
Comment (hannahhenderson) [inline server/resources/place/placeModel.js]: Per our conversation, STET.

File: server/resources/place/placeRoute.js [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+        .then(place => {
+          return place.addArt(art);
+        })
+        .then(() => {
+          res.send('updated');
+        });
+    });
 });
 
 router.get('/:id', (req, res) => {
```
Comment (hannahhenderson) [inline server/resources/place/placeRoute.js]: Please pull the photos (below) out of the git history, thank you!

**Comment by hannahhenderson** [comment]
Aside from the comments, LGTM.

**Pull request merged**